### PR TITLE
feat: Add default mapping to open deals after export to Pipedrive

### PR DESCRIPTION
**Pull request by maximumstock**
Adds a default mapping for Pipedrive exports to link to the exported deal.

### CODE REVIEW

File: Pipedrive.json [hunk 1]
```diff
@@ -237,6 +237,11 @@
             "detailsTemplate": "{{ type }}"
           }
         ]
+      },
+      "config": {
```
Comment (NetWin) [inline Pipedrive.json]: the same property probably needs to be set in the `visitreport` part of the mapping right? could you please add it there too?

Comment (maximumstock) [reply Pipedrive.json]: done 👍

**Comment by zogga007** [comment]
will be done in https://github.com/snapADDY/snapaddy-mapping-samples/pull/13

**Comment by maximumstock** [comment]
Hab mich mal unassigned hier :D

**Pull request closed without merging**